### ISSUE TITLE
Migrate from Goerli to Sepolia on Staging

### ISSUE THREAD

**Issue by aminlatifi** [Giveth/GIVeconomy#904]
Goerli testnet sunsets Apr 17.
On the staging env, we use Goerli testnet to mimic Ethereum Mainnet. We have to migrate to Sepolia on staging if we want it to look legit.

**Comment by laurenluz** [comment]
curious - does this mean that Goerli will just cease to exist? or it will just start to get buggy/not maintained? @aminlatifi

**Comment by aminlatifi** [comment]
> Refere

Most validators has been shut down. The chain may continue in an instable fashion, but we cannot count on it as etherscan is not supporting it anymore.

**Comment by jainkrati** [comment]
we did move to sepolia right @mohammadranjbarz @aminlatifi ? can I mark this closed

**Comment by mohammadranjbarz** [comment]
> we did move to sepolia right @mohammadranjbarz @aminlatifi ? can I mark this closed

Yes sure

**Comment by aminlatifi** [comment]
No, we have only moved Optimism Goerli to Optimism Sepolia. Goerli network on staging mimics Ethereum Mainnet in production and we haven't moved it.

**Comment by mohammadranjbarz** [comment]
> No, we have only moved Optimism Goerli to Optimism Sepolia. Goerli network on staging mimics Ethereum Mainnet in production and we haven't moved it.

Oh you are right
